frontend/packages/dev-console/src/components/formik-fields/DropdownField.tsx

```diff
@@ -14,14 +14,16 @@ const DropdownField: React.FC<DropdownFieldProps> = ({ label, helpText, ...props
       controlId={`${props.name}-field`}
       validationState={getValidationState(error, touched)}
     >
-      <ControlLabel className={cx({ 'co-required': props.required })}>{label}</ControlLabel>
+      {label && (
+        <ControlLabel className={cx({ 'co-required': props.required })}>{label}</ControlLabel>
+      )}
       <Dropdown
         id={`${props.name}-field`}
-        {...field}
-        {...props}
+        selectedKey={field.value}
         dropDownClassName={cx({ 'dropdown--full-width': props.fullWidth })}
         onChange={(value: string) => setFieldValue(props.name, value)}
         onBlur={() => setFieldTouched(props.name, true)}
+        {...props}
```

Comment:
Earlier API was that the parent would send `selectedKey` to `DropdownField` but i realised that it was redundant and using `field.value` instead is better option. I added `...props` just so that old APIs don't break. I didn't update that just to keep the PR small and concise.